src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -74,7 +76,9 @@ export function stringifyQueryObject(results: QueryResults) {
   const {query, ...tags} = results;
 
   const stringTags = flatMap(Object.entries(tags), ([k, values]) =>
-    values.map(tag => `${k}:${/[\s\(\)]/g.test(tag) ? `"${tag}"` : tag}`)
+    values.map(
+      tag => `${k}:${/[\s\(\)\\"]/g.test(tag) ? `"${escapeDoubleQuotes(tag)}"` : tag}`
```

Comment:
I feel like this function has a leaky interface as you use its pattern outside the function. Ideally the escapeDoubleQuotes() function would be in this module as this is the only place it is used and not require its pattern to be redefined. But this will be safe to move forward with.

Reply:
I've since moved `escapeDoubleQuotes()` exclusively to this function.